samples/net/sockets/echo_client/overlay-cc2520.conf

```diff
@@ -1,7 +1,6 @@
 CONFIG_NET_L2_IEEE802154=y
 
 CONFIG_IEEE802154_CC2520=y
-CONFIG_IEEE802154_CC2520_AUTO_ACK=y
```

Comment:
this basically means we never build with this overlay... @tbursztyka ?

Reply:
Dunno, these overlays have been recently introduced on socket echo apps.

Reply:
> Dunno, these overlays have been recently introduced on socket echo apps.

If they were recently introduced on socket echo apps, it only means they were ported over from non-socket echo apps.